source/CardGames.py

```diff
@@ -155,4 +157,57 @@ def dealCards(self, numCards: int, players: "list[Player]"):
 
   def resetDeck(self):
     self.deck.reset()
-    self.deck.shuffle()
\ No newline at end of file
+    self.deck.shuffle()
+
+#This will hopefully be my function location. -Logan C.
+def DisplayPlayerInfo(PlayerIn):
+  print(PlayerIn.name, "Has $", PlayerIn.money, '$ left')
+
+#def slowPrint(textIn):
+#  for 
+
+#This function makes messages to the user better looking
+def TextFormat(text1):
+  formattedText = textwrap.fill(text1, 50)
+  return formattedText
+
```

Comment:
Looks good I really like the design with the fishes

Reply:
lol sounds good bruv